runner/update-status

```diff
@@ -16,18 +16,27 @@ if [[ ${RUNNER_STATUS_UPDATE_HOOK:-false} == true ]]; then
     token=$(cat ${serviceaccount}/token)
     phase=$1
     message=${2:-}
-    repository=${3:-}
-    owner=${4:-}
-    workflow=${5:-}
-    [[ -n "$repository" ]] && repo=${repository#"$owner/"}
 
     data=$(jq -n --arg phase "$phase" \
       --arg message "$message" \
-      --arg repository "$repository" \
-      --arg owner "$owner" \
-      --arg repo "${repo:-}" \
-      --arg workflow "$workflow" \
-      '.status.phase = $phase | .status.message = $message | .status.repository = $repository | .status.owner = $owner | .status.repo = $repo | .status.workflow = $workflow')
+      --arg workflow_repository "${GITHUB_REPOSITORY:-}" \
```

Comment:
I tried to name the arg name after the status field path for consistency. For example, this is `workflow_repository` because the corresponding status field path is `status.repository`.

Reply:
With the new change thes new syntax makes sense